userguide/storage.rst

```diff
@@ -1803,6 +1803,14 @@ Hot spares can be added to a volume during or after creation. On
 %brand%, hot spare actions are implemented by
 `zfsd(8) <https://www.freebsd.org/cgi/man.cgi?query=zfsd>`__.
 
+Add a spare by going to
+:menuselection:`Storage --> Volume Manager`.
+Select the volume to extend from the :guilabel:`Volume to extend` drop
+down. Choose a disk from the list of :guilabel:`Available disks` and
+click **+** to add that disk to the volume. Select :literal:`spare` in
```

Comment:
GUI element should be marked up that way, and contents of fields are in single stars:
```suggestion
click :guilabel:`+` to add that disk to the volume. Select *spare* in
```